roles/satellite-clone/tasks/restore.yml

```diff
@@ -18,7 +18,7 @@
 
 - name: Restore mongo data
   command: "tar --selinux --overwrite -xf {{ backup_dir }}/mongo_data.tar.gz -C /"
-  when: standard_backup
+  when: rhel_migration or standard_backup
```

Comment:
On second thought, I like this to be simple human readable format and like this to be directly correlating to https://github.com/RedHatSatellite/satellite-clone/blob/master/docs/satellite-clone.md#prerequisites for easy maintenance in future.

So I like to keep it the same way `rhel_migration or standard_backup`